release-team/release-team-selection.md

```diff
@@ -0,0 +1,102 @@
+- former Release Team members
+
+Bear in mind that these are suggestions based on precedent and a Release Team Lead may be nominated by any Release Team member, past or present.
+
+_The new Release Team Lead can be selected via lazy consensus of the current Release Team members._
+
+### Release Team Lead Shadow
+
+Same as [Release Team Lead](#release-team-lead).
+
+### Shadows
+
+Every Release Team role should seek to accommodate a set of role shadows. This creates a new avenue for code and non-code contributors alike to contribute to the project. Additionally, it seeds future release cycles with new leaders.
+
+While there isn't a strict restriction on the amount of shadows, we've found three shadows per role to be a reasonable upper bound. However, as shadowing is intended to be a mentor / mentee journey, it is important that role leads only accept an amount of shadows that they feel that can reasonably mentor within the scope of a release cycle.
+
+If there are more contributors interested in shadowing once hitting that upper bound, we welcome them to sit in on Release Team meetings in preparation for shadowing in a future release cycle.
+
+In Kubernetes 1.14, we begun handling shadow selection using a questionnaire, as opposed to the first-come, first-served GitHub issue approach that we had adopted during previous release cycles ([1.12](https://github.com/kubernetes/sig-release/issues/167), [1.13](https://github.com/kubernetes/sig-release/issues/280).
+
+Moving to a [questionnaire](#sample-questionnaire) / survey format will allow us to capture more of the information required to vet potential shadows and well as begin to generate metrics around some of the Release Team processes.
+
+Questionnaire creation will be facilitated by SIG Release Chairs or a delegated coordinator to ensure consistency across release cycles.
+
+Following submissions, the questionnaires will be reviewed by role leads of the incoming Release Team.
+
+Volunteers that meet the requirements of the respective role will be contacted by those leads via some convenient mechanism (Slack, video chat, etc.) to further discuss.
+
+After vetting the volunteers for their roles, role leads should make a final decision on selected shadows with the incoming Release Team Lead.
+
+
+### Sample Questionnaire
+
+Thank you for volunteering for the Kubernetes 1.14 Release Team! 
+
+This questionnaire is meant to learn a little bit more about you, your journey with Kubernetes, your understanding of the Release Team processes and procedures, as well as to determine if you're a good fit for the current Release Team. Please note that you are expected to be inexperienced in some areas, as a role shadow is a mentorship opportunity.
+
+Release Team meetings/events have traditionally been US timezone-favoring (US Pacific, to be exact) due to personnel/tooling restrictions. While we're making efforts to change that, please be aware of this while completing the survey.
```

Comment:
To which restrictions are you referring here?

I know there were historical biases to for example a certain company's employees.  But aren't we waaay past that?

Besides perhaps the need to mostly asynchronously be preparing such that the right Google employee(s) are available to create RPMs at a specific time, I don't see blockers on participation in the release team.  For example, say the release team lead and branch manager lead are in China.  So long as Google holds the keys for a part of building, I would think it's on Google to staff or automate that builder role so as to not block the community's cadence.  Aside from that extreme example, if we're going to do formal releases at 3pm Pacific, we need coverage at that time in all roles.  But that's a choice right?  We could do releases at 3pm China.

Reply:
Tweaked after discussing with Tim offline.